### PR TITLE
Fix binder

### PR DESCRIPTION
**Pull request by trungleduc**
I switched to docker in order to use a base image containing already FreeCAD, it helps reduce the image build time on Binder.

### CODE REVIEW

File: binder/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,19 @@
+FROM trungleduc/jupytercad-env:0.1.1
```
Comment (martinRenou) [inline binder/Dockerfile:1]: Should we move this docker image under the jupytercad repo?

Comment (trungleduc) [reply binder/Dockerfile:1]: You mean the docker file or creating `jupytercad` account on the docker hub?

Comment (martinRenou) [reply binder/Dockerfile:1]: I mean the Dockerfile that was used for publishing `trungleduc/jupytercad-env` on Dockerhub?

Comment (trungleduc) [reply binder/Dockerfile:1]: ah ok, where should I put it?

Comment (martinRenou) [reply binder/Dockerfile:1]: Maybe under a `docker` directory on this repo?

Comment (martinRenou) [reply binder/Dockerfile:1]: For the account on docker hub, we could probably use the https://hub.docker.com/u/quantstack one

Comment (trungleduc) [reply binder/Dockerfile:1]: Do you have the right to push images to this account?

**Comment by martinRenou** [comment]
I don't see the JupyterCAD extension working on Binder, though it seems to be enabled:
 
Screenshot from 2023-01-05 10-44-29

**Comment by trungleduc** [comment]
That's weird, it worked till yesterday before changing the base image.

**Pull request merged**